statusCodes.js

```diff
@@ -0,0 +1,20 @@
+var NullValue = {
```

Comment:
these variable names shouldn't start with capital letters, unless the variable contains a class

Reply:
also, if you're using `let` and `const`, then you should use them consistently throughout your files